packages/docs/src/css/custom.css

```diff
@@ -13,7 +13,7 @@
   --ifm-color-primary-light: #a36397;
   --ifm-color-primary-lighter: #a86a9c;
   --ifm-color-primary-lightest: #b581ab;
-  --ifm-font-size-base: 90%;
+  --ifm-font-size-base: 16px;
```

Comment:
I just checked, and the current size using `90%` is `14px`.  If I remember correctly, we reduced font size to accommodate the FHIR resource pages, which can have big tables with lots of data. 

I'm ok to tweak font size.  I just want to make sure we checked all of the impacts.